gcn/notices/chime/retraction.example.json

```
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/chime/frb.schema.json",
+  "alert_type": "retraction",
+  "id": "FRB 20231013B",
```

```suggestion
  "id": ["FRB 20231013B"],
```

Same reason as previous example